### DISCUSSION TITLE
Ability to link to external PHP versions

### DISCUSSION THREAD

**Discussion by joshbenham** [beyondcode/herd-community#577] [Q&A]
Currently the company I work for has a lot of legacy applications that run on PHP7.0. These applications will literally never be updated to a newer PHP version as they ... "don't need to be".

I was wondering if it is possible to link to an external PHP version. On my current machine I have php7.0 installed through Homebrew and was wondering if its possible to use that.

As I will not need to leverage composer for this Project hoping that simplifies it. Or is there a way to tell NGINX for this site to run a different PHP version?

Something along the lines of

```bash
$ herd use /usr/bin/php7.0
```

Currently I am using ddev (docker) to run the old websites but I would really just love to blow away docker completely.

**Comment by thomthom** [comment]
I'm wondering the same. I have a site that runs 7.1, and I am planning on upgrading. But I need to start with what I already have and work myself up with the Laravel and PHP versions. (Yes, it's quite a few years old site.)
